addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -2499,7 +2499,13 @@ export class OdooEditor extends EventTarget {
             const selection = this.document.getSelection();
             if (selection.isCollapsed) {
                 ev.preventDefault();
-                this._applyCommand('oDeleteBackward');
+                const inputEvent = new InputEvent('input', {
+                    inputType: 'deleteContentBackward',
+                    data: null,
+                    bubbles: true,
+                    cancelable: false,
+                });
+                this._onInput(inputEvent);
```

Comment:
About the commit message. Do we have a specific case in mind ? It would be great if we could explain the issue with a specific case that doesn't work without this change.